jaeger-jaxrs2/src/main/java/com/uber/jaeger/filters/jaxrs2/ClientFilter.java

```diff
@@ -43,10 +45,12 @@
     private final Tracer tracer;
     private final TraceContext traceContext;
     private final Logger logger = LoggerFactory.getLogger(ClientFilter.class);
+    private final Metrics metrics;
 
-    public ClientFilter(Tracer tracer, TraceContext traceContext) {
+    public ClientFilter(Tracer tracer, TraceContext traceContext, StatsFactory statsFactory) {
```

Comment:
I suggest adding another constructor that allows null as 3rd argument, and delegating to it from the existing one, for backwards compatibility